libavogadro/src/extensions/yaehmop/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+include_directories(${CMAKE_CURRENT_BINARY_DIR})
+
+set(LINK_LIBS avogadro)
+set(PLUGIN_LABEL extensions)
+set(PLUGIN_TARGET extensions)
+
+set(yaehmopextension_SRCS
+  yaehmopbanddialog.cpp
+  yaehmopextension.cpp
+  yaehmopout.cpp
+)
+
+set(yaehmopextension_UIS
+  yaehmopbanddialog.ui
+)
+
+avogadro_plugin_nogl(yaehmopextension
+  "${yaehmopextension_SRCS}";
+  "${yaehmopextension_UIS}"
+)
+
+# Download Yaehmop executable
+# Let's set the name. Windows likes to add '.exe' at the end
+if(WIN32)
+  set(YAEHMOP_NAME "yaehmop.exe")
+else(WIN32)
+  set(YAEHMOP_NAME "yaehmop")
+endif(WIN32)
+
+if(NOT EXISTS ${CMAKE_BINARY_DIR}/bin/${YAEHMOP_NAME})
```

Comment:
There needs to be a CMAKE option for USE_SYSTEM_YAEHMOP and if false, download it. Otherwise, Linux packaging will be unhappy.